package/src/components/AutoCompleteInput/AutoCompleteInput.tsx

```diff
@@ -425,12 +423,6 @@ const AutoCompleteInputWithContext = <
       testID='auto-complete-text-input'
       value={text}
       {...additionalTextInputProps}
-      onBlur={(event) => {
-        if (additionalTextInputProps?.onBlur) {
-          additionalTextInputProps?.onBlur(event);
-        }
-        setShowMoreOptions(true);
```

Comment:
good riddance,

my also remove getting `setShowMoreOptions` from context in this component as well

Reply:
There are other components overriding the state like InputButtons, InputGiphySearch so probably here it will be an issue.